resource_booking/templates/portal.xml

```diff
@@ -545,6 +545,7 @@
             </t>
             <!-- Scheduling form -->
             <t t-call="portal.portal_record_layout">
+                <t t-set="classes" t-value="'resource_booking_portal_schedule'" />
```

Comment:
This only adds extra classes: https://github.com/odoo/odoo/blob/15.0/addons/portal/views/portal_templates.xml#L323

Reply:
Yes, but what happens if other module set the same variable? Yours will be overwritten, or vice versa.

Reply:
If the problem is to have a specific selector, the view is built from this module, so we can put any other needed class/id.

Reply:
I've been testing, in particular I've set a red background class in another call to the same template and each call maintains its own custom classes. I think that what @victoralmau proposes is quite coherent having the option to modify the class. In this way the class is applied at the call level, so if from another place they make a call to the same template and modify the classes it won't affect this particular call but it won't propagate to other calls either. That is to say, what is established in the t-set will have repercussions on this specific call, it can neither be modified from outside and cannot be propagated to other calls.

Reply:
OK then.